MetaMask/metamask-extension
# feat: add matchain logo svg

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27792?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to add network
2. Add Matchain
3. Matchain should have a logo
4. Native token should have a logo same with BSC

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
